### PR TITLE
Building with Responsive Design.

### PR DESCRIPTION
**Pull request by AdesojiCodeMaster**
The Project Title:**The TNW-PAGE**

Authors: **Adesoji** and **Carlos**

[The Project/original link](https://thenextweb.com/)

[Live Demo Link](https://rawcdn.githack.com/carlos-ssh/tnw-page/4b9caa9fa415f05ce5be3b0cdcc1569e1bf87ea9/index.html).

**Note;** Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images,  the texts, and sometimes the design on the original page keep on changing per time.

### CODE REVIEW

**Comment by AdesojiCodeMaster** [comment]
The Project Title:**The TNW-PAGE**

Authors: **Adesoji** and **Carlos**

[The Project/original link](https://thenextweb.com/)

[Live Demo Link.](https://rawcdn.githack.com/carlos-ssh/tnw-page/2816a803b7ba4374a09b8c8d417a53d30b8b9935/index.html)

**Note;** Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images, the texts, and sometimes the design on the original page keep on changing per time.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Chnages required :recycle: 

### Hi @AdesojiCodeMaster and @carlos-ssh 

### Your project is getting better and better but the there is some changes to be made:

- Don't add unnecessary files to the repository, you should add them to .gitignore file and delete them from the github repository.
Screenshot from 2020-06-25 12-01-27
- For media queries, your page does not behave similar to the original, please refactor your code so the most of the changes are around the following: - medium: min-width around 768px and not 640 px as you are doing currently.
- All the sections should take the full screen and should be aligned with each other:
clone
Screenshot from 2020-06-25 12-13-28
Screenshot from 2020-06-25 12-13-45
Original:
Screenshot from 2020-06-25 12-16-03
- In small width the page is not responsive
Screenshot from 2020-06-25 12-16-37

### once done with the changes, please ask for another code review

**Comment by carlos-ssh** [comment]
**Hello, TSE's reviewer's team. thanks for your support giving continuity to the changes requested. We fixed these next points to this project:**

- We removed unnecessary files from the repository.
- Changed the media query to min-width around 768px.
- Aligned all sections from full screen.
- Fixed issues in responsive design.

We leave the live demo link here: >
[https://rawcdn.githack.com/carlos-ssh/tnw-page/092b8ed37b1af97670d2813d729c7fd064c14ee6/index.html](url)

_Thanks again for the support_

**Happy Coding!**

Carlos & Adesoji.

**Comment by sumancrest0001** [comment]
**Status: changes required
Great work TEAM :clap:** You have fixed most of the issues addressed by the previous TSE and the page looks responsive enough.  But there are some minor issues still remaining and we need to fix them to get :+1: Let's make the following changes.
1. Please align all the sections at the center as on the original page. Especially, latest news section, apps & gear section, and the latest deal section.
2. Give some margin to the **latest news** title as it is overlapping with the main article on a mobile screen.
3. On a mobile screen, the second image of apps, gears section is not displaying properly. Please fix it.
Currently :point_down: 
Screenshot from 2020-06-26 07-43-36

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the requested changes and fill up another review form.
If you have any queries regarding the review, DM me on slack Suman Shrestha
Happy coding:slightly_smiling_face:**

**Comment by AdesojiCodeMaster** [comment]
The Project Title:  **The TNW-PAGE**

Authors:  **Adesoji** and **Carlos**

**The recommendations have been fixed, kindly check it out. Thank you.**

[The Project/original link](https://thenextweb.com/)

 [Live Demo Link.  ](https://rawcdn.githack.com/carlos-ssh/tnw-page/d6869adb73120c51f02d8c7759247140eaa1d79c/index.html)

**Note;** Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images, the texts, and sometimes the design on the original page keep on changing per time.

File: assets/style/style.css [hunk 1]
```diff
@@ -0,0 +1,2836 @@
...
+  font-size: 11px;
+  left: 10px;
+  padding-top: 10px;
+  position: relative;
+  width: 200px;
+  top: 10px;
+}
+
+.latest-div7 a {
+  bottom: 15px !important;
```
Comment (elmejdki) [inline assets/style/style.css]: ### please remove all the !important keywords and try to override style based on CSS selector specificity, here is a good [article](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) talking about it

File: assets/style/style.css [hunk 2]
```diff
@@ -0,0 +1,2836 @@
...
+}
+
+.funding-img-cont {
+  bottom: 150px;
+  display: grid;
+  gap: 25px;
+  grid-template-columns: repeat(4, auto);
+  margin: 0 50px 0 10px !important;
+  position: relative;
+  width: 100vw !important;
```
Comment (elmejdki) [inline assets/style/style.css]: here also remove important

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Grea job guys :clap:  you have made a really good clone :tophat:.
# but in order to get approved you need to make some changes :hammer: :hammer_and_pick: :wrench: :toolbox: 
## status: require changes. :recycle: 

- [ ] please remove overflow-x attribute in this selector cause on the mobile view we can scroll down or up:
image

- [ ] After removing the overflow-x: hidden property you will get this space on the right please fix it if you got stuck let me know I know how to fix it:
image

- [ ] Fix the subtitles of these articles, on the mobile view:
### clone:
image

### original:
image

- [ ] use the space on the right of these articles don't leave them empty:
### clone:
image
### original:
image
- [ ] you have used !important several times in your CSS please remove them all as suggested in the comment below.

### Please fix all the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you got stuck or you haven't question don't hesitate to contact me on slack_

**Comment by AdesojiCodeMaster** [comment]
**The Project Title:** **The TNW-PAGE**    

**Authors:** **Adesoji** and **Carlos**

**The recommendations pointed out have been fixed, kindly check it out. Thank you.**

[The Project/original link](https://thenextweb.com/)

[Live Demo Link.](https://rawcdn.githack.com/carlos-ssh/tnw-page/cca6f8c1a5a479861a72e11cf7360da869947bf7/index.html)

**Note;** Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images, the texts, and sometimes the design on the original page keep on changing per time.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello team 👋! You have done an incredible job making this project and applying the changes required by the previous reviewers 👏🙌.

### But I see that you have missed one minor change as well.

- The horizontal scrollbar still remains. Adding `overflow-x: hidden;` is not a solution. I would suggest you reduce the width of the elements.

Annotation 2020-06-28 195528


### In addition to the above:

- In the `top 3 articles` section, the articles would be wrapped in `article` tags. The headings inside them should be wrapped around `h1/h2` tags. Please make sure of that.

- The social icons in the footer are not like the original. Change their styles so that they look exactly like the original.

#### Clone
image

#### Original
image

Apply these changes and request another code review.

Happy coding 👨‍💻

**Comment by AdesojiCodeMaster** [comment]
**The Project Title: The TNW-PAGE**    

**Authors:**   **Adesoji**  and **Carlos**

**Fixed the issues pointed out by the last TSE, kindly check it out. Thank you.**

 [The Project/original link](https://thenextweb.com/)

  [Live Demo Link.](https://rawcdn.githack.com/carlos-ssh/tnw-page/89a69894df9fd9ec57e87d428eede7e4327a8830/index.html)

**Note;** Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the **images,** the **texts,** and sometimes the **design** on the original page keep on changing per time.

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES  REQUIRED :recycle: 
### HELLO :wave:  I AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT :eye: 

**GREAT WORK IMPLEMENTING ALL THE CHANGES AS REQUESTED BY THE PREVIOUS TSE** :clap: 

However, before we continue let's make the following changes. :arrow_down: 

- [ ] Kindly fix the responsiveness issues in some parts of your project. Kindly check your footer on the mobile view some icons are being hidden.
footer responsiveness


- [ ] Also the latest news section has some responsiveness issues kindly check how the images are being hidden.
check from 770 px

improve the responsiveness


- [ ] Also just like suggested by the previous TSE only use `overflow-x : hidden` when it's absolutely necessary. First, make sure your images text and anything you have in your webpage fit correctly even without it


### if you have any question kindly reach me via slack @peterrobert


### HAPPY CODING

**Comment by AdesojiCodeMaster** [comment]
**The Project Title:** **The TNW-PAGE**

**Authors:** **Adesoji** and **Carlos**

**Fixed the issues pointed out by the last TSE, kindly check it out. Thank you.**

**To also address the last comment  made by the last TSE, we did not use 'overflow-x: hidden;' property initially but when one of the TSEs pointed it out that there was overflowing that was why we put it to avoid issues, although on our screens no overflow is showing but due to the fact that we all have different screen sizes and to prevent any other unseen overflows.**

 [The Project/original](https://thenextweb.com/)

 [Live Demo Link. ](https://rawcdn.githack.com/carlos-ssh/tnw-page/86c1bc733d3cb063f788f1d9ef177aca23869701/index.html)

**Note;** Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images, the texts, and sometimes the design on the original page keep on changing per time.

File: assets/style/style.css [hunk 3]
```diff
@@ -0,0 +1,3066 @@
...
+    bottom: 50px;
+    left: 110px;
+    position: relative;
+  }
+
+  #jabs,
+  #kontak {
+    position: relative;
+    left: 35px;
+  }
```
Comment (andresporras3423) [inline assets/style/style.css]: Don't overuse ID selectors

File: assets/style/style.css [hunk 4]
```diff
@@ -0,0 +1,3066 @@
...
+  grid-row: 1;
+  height: 30px;
+}
+
+.apps-txt2 {
+  display: grid;
+  grid-column: 3;
+  grid-row: 1;
+  height: 30px;
+}
```
Comment (andresporras3423) [inline assets/style/style.css]: use a class for the common styles in the three selectors

**Review by andresporras3423 — COMMENTED** [review]
### Status: Changes Required :recycle:
**Hello @AdesojiCodeMaster @carlos-ssh** 
Good job with the improvements for this project :muscle:  
Your page really looks like a clone of the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just a few details to improve before you can go ahead:
- [ ] Don't overuse ID Selectors
- [ ] If you have common styles in different selectors you can create a class for the common styles, that way you don't repeat yourself and can make your code more reusable.

### OPTIONAL

- [ ] Your main tag shouldn't contain the header and the footer.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by AdesojiCodeMaster** [comment]
**The Project Title:** **The NW-PAGE**

**Authors:**  **Adesoji** and **Carlos**

**Fixed the issues pointed out by the last TSE, kindly check it out. Thank you.**

[The Project/original ](https://thenextweb.com/)

[Live Demo Link. ](https://rawcdn.githack.com/carlos-ssh/tnw-page/5b1e3ce716a4091c1d57a417db11d2d3a12aa6ba/index.html)

Note; Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images, the texts, and sometimes the design on the original page keep on changing per time.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status require changes :recycle: 
## Hello guys, it's me again :smiley:, great job you are almost there :clap: :clap: thank you for making the required change, but in order to make your project approved you need to make just one change :hammer:.

### :warning: I remember that I have told you to remove the overflow-x: hidden; CSS property but you didn't, I see that there is a better way to solve the issue that you are facing, by just minimizing the size of other elements that made the horizontal scroll show, please fix that and if you are stuck don't hesitate to contact me on slack I know how to solve it.

### Please make the required changes before you request another code review.
## Happy Coding :tophat:

**Comment by carlos-ssh** [comment]
The Project Title: The NW-PAGE

Authors: Adesoji and Carlos

Fixed the issues pointed out by the last TSE, kindly check it out. Thank you.

[The Project/original](https://thenextweb.com/)

[Live Demo Link.](https://rawcdn.githack.com/carlos-ssh/tnw-page/8126f3a11801d672e64811c4d3f32d20b99966a5/index.html)

Note; Kindly note that the images, the texts, and the design on this project are what we have as the time we started the project, this is because the images, the texts, and sometimes the design on the original page keep on changing per time.

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey Team  :wave:
-
Good job with the changes! :tada: Keep it up. 
We still do have some few changes to carry out before we proceed (see review comment). 
Attention to detail would go a long way in this project, kindly let us adhere to that
##
## Main section
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/carlos-ssh/tnw-page/feature/index.html)|
|:---:|:---:|
|source|clone|
- [ ] The comparison between the two shows some differences lets work on them
- - [x] The logo should be seen at the top-left corner of the article
- - [x] Add the missing texts and respective dates on the top 3 articles and style them appropriately.
- - [x] The top heading of the `latest news`, `latest fundings`, and `latest deals` sections are out of place. Reposition them.   

## Footer section
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/carlos-ssh/tnw-page/feature/index.html)|
|:---:|:---:|
|footer|footer-clone|
- [x] Improve on the styling of the footer's social icons. they should match their counterparts on the source page.  
##

- [ ] I noticed there was a lot of repeated overriding of class styles in the CSS. This goes against the DRY principle of programming. Let's ensure  that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).
- [ ] The page has an overbearing horizontal scroll in all screen sizes. Let's fix that as well.
- [ ] The page is also not responsive in small screens. Let's work on that as well. 
##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by carlos-ssh** [comment]
**Hello TSE's team please check the project we worked on the last changes requested.**

-  Fixed the logo to the top-left corner of the article.
-  Added the missing texts and respective dates.
-  Fixed the Footer section. 
- We cleaned the CSS file avoiding bad practices.
- We fixed the horizontal scroll in all screen sizes.
- We worked fixing issues that were affecting the responsiveness in small screens.

This is the [Live Demo Link](https://rawcdn.githack.com/carlos-ssh/tnw-page/df423285c108a2dec068828a2b2cb963b3378808/index.html)

Link of the [Original Project](https://thenextweb.com/)

_Thanks for your time and patience._

**_Happy Coding!_**

**Carlos & Adesoji.**

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Hi, it's me again 😄 . Good job making all the changes 👏! Kudos for removing the horizontal scrollbar 💯 ! You are almost there.

- As mentioned by the previous reviewer, attention to detail goes a long way. Let's make the latest deals section look like the original.

#### Clone
Annotation 2020-07-07 091655

#### Original
image

- Let's add some `line-height` to the headings of the top 3 articles to prevent the lines from encroaching on each other.
image

- Let's make each box of latest deals have the same width in tablet view.

#### Clone
image

#### Original
image

- Let's decrease the height of each article in the top 3 articles on the mobile view to match the original.

#### Clone
image

#### Original
image

- Let's increase the logo size on the mobile view.

#### Clone
image

#### Original
image


Apply these changes and request another review.


Happy coding 👨‍💻

File: style.css [hunk 5]
```diff
@@ -1,4 +1,795 @@
...
+}
+
+.beige2 {
+  background-color: #fffaf8;
+  height: 45px;
+}
+
+.funding-grid-item4 {
+  display: none;
+}
```
Comment (andresporras3423) [inline style.css:317]: Same style definition of the selector on line 133. Group selectors or create a new class.

File: index.html [hunk 6]
```diff
@@ -1,11 +1,589 @@
...
+
+                    <article>
+                        <div class="funding-title-div">
+                            <a href="#">
+                                <h2 class="funding-header">Latest funding rounds on Index.co ></h2>
+                            </a>
+                            <ul class="funding-anchors">
+                                <li> <a href="#" style="color: #f42;"> See more</a></li>
+                                <li> <a href="#" style="color: #f42;"> Get updates by email </a></li>
+                                <li> <a href="#" style="color: #f42;"> Add info</a></li>
```
Comment (andresporras3423) [inline index.html:203]: Don't use inline styles in your html

**Review by andresporras3423 — COMMENTED** [review]
Sorry, by mistake I added the review here. But remember not to open a new pull request for each review. Use always the same pull request.

**Comment by carlos-ssh** [comment]
@andresporras3423  **Don't worry... I need that you specify me the changes that you request in this line:**

- I think you need to update the live demo. In your CSS you have the correct breakpoint but your demo uses a different one (with 640px for small screen size).

_Thanks._

**Carlos**

**Pull request closed without merging**